client/src/App.jsx

```diff
@@ -0,0 +1,15 @@
+import React, { useState } from "react";
+import io from "socket.io-client";
+import Chat from "./Chat/Chat";
+
+const socket = io.connect("http://localhost:8080");
+
```

Comment:
Thinking that this might look cleaner if the http address was imported from process.env or a hidden config file. Not a big priority.